README.md

```diff
@@ -9,15 +9,16 @@
 - We are releasing **[Stable Video 4D (SV4D)](https://huggingface.co/stabilityai/sv4d)**, a video-to-4D diffusion model for novel-view video synthesis. For research purposes:
     - **SV4D** was trained to generate 40 frames (5 video frames x 8 camera views) at 576x576 resolution, given 5 context frames (the input video), and 8 reference views (synthesised from the first frame of the input video, using a multi-view diffusion model like SV3D) of the same size, ideally white-background images with one object.
     - To generate longer novel-view videos (21 frames), we propose a novel sampling method using SV4D, by first sampling 5 anchor frames and then densely sampling the remaining frames while maintaining temporal consistency.
+    - You can run the community-build gradio demo locally by running `python -m scripts.demo.gradio_app_sv4d`.
```

Comment:
Change text to "- To run the community-build gradio demo locally, run `python -m scripts.demo.gradio_app_sv4d`."

Reply:
fixed